### PR TITLE
VIH-6590 VHO copies telephoneconferenceId to clipboard

### PR DESCRIPTION
**Pull request by virensareen**
### JIRA link (if applicable) ###
VIH-6590

### Change description ###
Added a telephone icon for VHO officer to copy the telephoneconferenceId

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-telephone-id/copy-telephone-id.component.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div>
+  <img
+    appTooltip
+    [text]="tooltip"
+    class="icons"
+    src="/assets/images/phone.png"
+    alt="copy joining by phone details to clipboard"
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-telephone-id/copy-telephone-id.component.html:7]: does this need to be a "C" ?

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-telephone-id/copy-telephone-id.component.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    constructor(private clipboardService: ClipboardService) {}
+
+    ngOnInit(): void {
+        this.tooltip = 'Copy joining by phone details to clipboard';
+    }
+
+    copyToClipboard() {
+        this.clipboardService.copyFromContent(this.telephoneId);
+        this.tooltip = 'Details copied to clipboard';
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-telephone-id/copy-telephone-id.component.ts:21]: If needed to copy again when tooltip opened would it still say 'Details copied to clipboard'?
I cant see anything resetting the state when the tooltip opens or closes?

**Review by levibutler — CHANGES_REQUESTED** [review]

**Review by levibutler — APPROVED** [review]

File: VideoWeb/VideoWeb.Contract/Responses/ConferenceForVHOfficerResponse.cs [hunk 3]
```diff
@@ -60,5 +60,10 @@ public class ConferenceForVhOfficerResponse
         /// Closed date time as UTC
         /// </summary>
         public DateTime? ClosedDateTime { get; set; }
+
+        /// <summary>
+        /// The telephone Id of the conference
+        /// </summary>
+        public string TelephoneConferenceId { get; set; }
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb.Contract/Responses/ConferenceForVHOfficerResponse.cs:67]: You've added new fields but not updated the assertions on the mappers

**Review by shaed-parkar — CHANGES_REQUESTED** [review]
please update the mapper tests

**Review by shaed-parkar — APPROVED** [review]
LGTM

**Pull request merged**